File: CS/App_Code/ThemeGroupModel.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Xml.Serialization;
using System.Web.UI;



public class ThemeGroupModel : ThemeModelBase {
    List<ThemeModel> _themes = new List<ThemeModel>();
    string _float;
    [XmlElement(ElementName = "Theme")]
    public List<ThemeModel> Themes {
        get { return _themes; }
        set { _themes = value; }
    }
    [XmlAttribute]
    public string Float {
        get { return _float; }
        set { _float = value; }
    }
}

File: VB/App_Code/ThemeGroupModel.vb
Option Infer On

Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Xml.Serialization
Imports System.Web.UI



Public Class ThemeGroupModel
	Inherits ThemeModelBase

	Private _themes As New List(Of ThemeModel)()
	Private _float As String
	<XmlElement(ElementName := "Theme")>
	Public Property Themes() As List(Of ThemeModel)
		Get
			Return _themes
		End Get
		Set(ByVal value As List(Of ThemeModel))
			_themes = value
		End Set
	End Property
	<XmlAttribute>
	Public Property Float() As String
		Get
			Return _float
		End Get
		Set(ByVal value As String)
			_float = value
		End Set
	End Property
End Class



